.github/workflows/npm-publish.yml

```
@@ -0,0 +1,18 @@
+name: Publish on NPM
+on:
+   release:
+      types: [created]
+jobs:
+   build:
+      runs-on: ubuntu-latest
```

Same version pinning suggestion:
```suggestion
      runs-on: ubuntu-20.04
```

---

I'd be pretty sweet if npm supported lts versions ala `packagename-latest-lts`